services/plugins/generic_plugins/requests.go

```diff
@@ -0,0 +1,56 @@
+package generic_plugins
+
+import (
+	"bytes"
+	"fmt"
+	"io"
+	"mime/multipart"
+	"os"
+	"strings"
+)
+
+func (genericPlugin *GenericPlugin) CustomEncoding(data []byte, isUpdating bool) (io.Reader, string, error) {
```

Comment:
It would be nice to factorize this with the `FileAndDataToMultipart` in plugins. The only difference is the name of parameters (file vs sourceCode) and the list of parameters to put inside (commands vs  name/description/type/language)

Reply:
Yes done